src/py/mat3ra/made/tools/modify.py

```diff
@@ -323,3 +319,42 @@ def condition(coordinate):
     return filter_by_condition_on_coordinates(
         material, condition, use_cartesian_coordinates=use_cartesian_coordinates, invert_selection=invert_selection
     )
+
+
+def add_vacuum(material: Material, vacuum: float = 5.0) -> Material:
```

Comment:
The selection of functions is a bit weird, usually we would follow a CRUD, get-set, or add-remove pattern. The `add` and `set` pattern does not fit either.

Reply:
For software. For material it makes sense to "add vacuum", "set vacuum" implies that there's no vacuum by default and we set the exact value. Maybe "remove vacuum" is more fair.